### PR TITLE
upgrade this package and dependencies

### PR DESCRIPTION
**Pull request by maxkoryukov**
Several features are included:

* upgrade all dependencies
* lint code with http://eslint.org/
* continuos build on [Travis CI](https://travis-ci.org/maxkoryukov/gulp-inline)
* online coverage: https://codecov.io/gh/ashaffer/gulp-inline
* badges in readme

Additionally:
* fix `basic.svg` issue (dup for ashaffer/gulp-inline#46)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "plugins": [
+        "standard",
+        "promise"
+    ],
+
+    "rules": {
+        "one-var": "off",
+        "space-before-function-paren": "off",
+
+        // remove this rules:
```
Comment (maxkoryukov) [inline .eslintrc.js]: I am not sure about code-style...

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  directories:
+    - node_modules
+
+before_install:
+  - npm config set spin false
+
+install:
+  - npm install
+  - npm install -g codecov
+  # - npm install -g bithound
```
Comment (maxkoryukov) [inline .travis.yml]: Uncomment lines with `bithound`, and their service will lint and check your code too (additionally to eslint)

File: test/main.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ var gulp = require('gulp')
 var fs = require('fs')
 var path = require('path')
 var inline = require('..')
-var transform = require('readable-stream/transform')
+var Transform = require('readable-stream/transform')
```
Comment (maxkoryukov) [inline test/main.js]: ESlint forced me to do this....

**Comment by maxkoryukov** [comment]
@ashaffer , I have fixed the conflicts;)

**Comment by maxkoryukov** [comment]
And, plz, could you publish new version on NPM??

**Comment by maxkoryukov** [comment]
@ashaffer are there a chance to merge this?

**Comment by ashaffer** [comment]
Ya, thanks. I'll do it now.

**Comment by ashaffer** [comment]
Bumped & published :). Thanks for all the work you've done lately.

**Comment by maxkoryukov** [comment]
@ashaffer  thank **you** for this cool gulp-plugin!!

**Pull request merged**